scripts/build

```
@@ -10,6 +10,14 @@ if [ "$(uname)" = "Linux" ]; then
     OTHER_LINKFLAGS="-extldflags -static -s"
 fi
 
+# add coverage flags if there is no tag and it's on master or a version branch like v1.3
+COMMIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+COMMIT_TAG=$(git tag --points-at HEAD | head -n 1)
+if [[ "$COMMIT_TAG" == "" ]] && [[ "$COMMIT_BRANCH" == master || "$COMMIT_BRANCH" =~ ^v[0-9]+\.[0-9]+$ ]]; then
+    COVER="-cover"
+    COVERPKG="-coverpkg=github.com/harvester/harvester/..."
```

just a nit: `COVER_FLAGS` might be better than `COVER` here.
```suggestion
    COVERPKG="-coverpkg=github.com/harvester/harvester/..."
    COVER_FLAGS="-cover $COVERPKG"
```

And just use `COVER_FLAGS` later. The rest look good! Thanks.

---

Updated it. Thanks.